### PR TITLE
EWS fixes

### PR DESCRIPTION
**Pull request by aviadl**
Added find folders command, support fetching mails from a predefined folder and different fixes to command outputs

### CODE REVIEW

File: Integrations/integration-Office_EWS.yml [hunk 1]
```diff
@@ -793,8 +870,14 @@ script:
...
         fetchArgs['target-mailbox'] = getDefaultTargetMailbox();
+        if (params.folder) {
+            fetchArgs['folder'] = lastRun.folderId;
+            fetchArgs['target-folder'] = replaceInTemplates(targetFolderTmpl,fetchArgs);
+        } else {
+         fetchArgs['target-folder'] = replaceInTemplates(defaultTargetTargetFolderTmpl,fetchArgs);
+        }
         fetchArgs.version = params.defaultServerVersion
-        raw = sendRequest(fetchTmpl, fetchArgs);
+        raw = sendRequest(fetchTmpl, fetchArgs.'Envelope.Body.FindItemResponse.ResponseMessages.FindItemResponseMessage.-ResponseClass');
```
Comment (asafshen) [inline Integrations/integration-Office_EWS.yml]: I think you accidentally wrote `.` instead `,` (between `fetchArgs` & `Envelope..`)

File: Integrations/integration-Office_EWS.yml [hunk 2]
```diff
@@ -1245,10 +1318,10 @@ script:
...
+    - contextPath: EWS.Items.Headers.-HeaderName
       description: Mail attachment header name
-    - contextPath: EWS.Items. Headers.#text
+    - contextPath: EWS.Items.Headers.#text
       description: Mail attachment text
     description: Get actual attachment item from email
   isfetch: true
-releaseNotes: "Support get attachment of an item(mail)"
\ No newline at end of file
+  releaseNotes: Added find folders command, support fetching mails from a predefined folder and different fixes to command outputs
```
Comment (asafshen) [inline Integrations/integration-Office_EWS.yml]: the space before `releaseNotes` fails it

**Comment by Spivaka** [comment]
are we adding a test PB or should I merge this?

**Comment by aviadl** [comment]
Yes, waiting for @michalgold on this

**Comment by asafshen** [comment]
@aviadl please review last commit

**Pull request merged**